test/DaprExtensionTests/Tests/DaprBindingTriggerTests.cs

```diff
@@ -120,11 +120,11 @@ static class Functions
             public static object DotNetBindingExpression([DaprBindingTrigger(BindingName = "%BindingName%")] string input) => input;
 
             [FunctionName("Add")]
-            public static string Sample([DaprServiceInvocationTrigger] JObject args, ILogger log)
+            public static string? Sample([DaprServiceInvocationTrigger] JObject args, ILogger log)
             {
                 log.LogInformation("C# processed a method request from the Dapr runtime");
 
-                double result = (double)args["arg1"] + (double)args["arg2"];
+                double? result = (double?)args["arg1"] + (double?)args["arg2"];
                 return result.ToString();
```

Comment:
Since you've declared `result` to be a nullable `double`, should we have a null check here to avoid a potential `NullReferenceException`?